beringei/tools/CMakeLists.txt

```diff
@@ -56,3 +56,4 @@ target_link_libraries(
 )
 
 add_subdirectory(grafana_read_service)
+add_subdirectory(plain_text_service)
```

Comment:
I prefer that you add this new functionality into the grafana_read_service. We can later rename service to a more generic name. This way, you wont need a copy of NotFoundHandler, Server.cpp, DateUtils & TestConnectionHandler.